src/Sylius/Bundle/AdminBundle/Resources/views/Order/Show/_shipment.html.twig

```diff
@@ -27,7 +27,11 @@
             <p>{{ shipment.tracking }}</p>
         </div>
     {% endif %}
-
+    {% if shipment.state == shipped %}
+        <div class="ui icon labeled tiny fluid button" style="margin-top: 10px;">
+            <i class="icon search"></i><a style="color: gray;" href="{{ path('sylius_admin_shipment_show', {'id': shipment.id}) }}" {{ sylius_test_html_attribute('shipment-show-button') }}>{{ 'sylius.ui.show'|trans }}</a>
+        </div>
```

Comment:
Did you check how it works? Because I guess that there will be a problem that only the text is clickable, not entire button